docs/deprecations.rst

```diff
@@ -1,6 +1,39 @@
 Deprecations
 ============
 
+streamlink 6.6.0
+----------------
+
+HTTPSession and HTTPAdapters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The module structure of the :class:`Streamlink <streamlink.session.Streamlink>` session implementation and related classes
+like the ``HTTPSession`` has been re-organized.
```

Comment:
`has` -> `have` since you are talking about multiple classes.

Reply:
No...

> The module structure ... has been re-organized